src/core/transform.ts

```diff
@@ -136,7 +136,7 @@ function generateStoryImport(
   return `
     ${renderFunction}
     export const ${id} = () => Object.assign({render: render${id}}, _sfc_main)
-    ${id}.storyName = '${title}'
+    ${id}.storyName = '${title}'${play ? `\n${id}.play = ${play}` : ''}
```

Comment:
Please put the addition in a new line.

Reply:
The problem here is that putting it in a new line will result in an additional blank line if no play function is supplied, this would require updating every test. Is that ok?

Reply:
Yes, that shouldn't be an issue.